firmware/esp32_ttgo/client/StreamClient.h

```diff
@@ -0,0 +1,22 @@
+#ifndef STREAMCLIENT_H
+#define STREAMCLIENT_H
+
+#include "../DisasterClient.h"
+#include "../DisasterServer.h"
+
+class StreamClient : public DisasterClient
+{
+  Stream *stream;
```

Comment:
What's the use case for this Stream client? Is it just a generic Arduino Stream, or do you have a specific purpose in mind?

Reply:
It's used currently used for the Serial console, since that's a Stream. In theory it could be used for other Streams like EthernetServer's.